web/src/app/schedules/ScheduleTZFilter.js

```diff
@@ -21,14 +22,15 @@ export function ScheduleTZFilter(props) {
     variables: { id: props.scheduleID },
   })
 
-  let label, tz
+  let label, tz, short
   if (error) {
     label = 'Error: ' + (error.message || error)
   } else if (!data && loading) {
     label = 'Fetching timezone information...'
   } else {
     tz = data.schedule.timeZone
-    label = props.label ? props.label(tz) : `Show times in ${tz}`
+    short = DateTime.fromObject({ zone: tz }).toFormat('ZZZZ')
```

Comment:
We can move the `let` keyword here since it doesn't need to be used outside this block. Also we should add a condition to not repeat if the `short === tz` (e.g., UTC)